sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml

```diff
@@ -84,7 +84,7 @@
     <dependency>
       <groupId>com.github.spotbugs</groupId>
       <artifactId>spotbugs-annotations</artifactId>
-      <version>4.8.0</version> <!-- {x-version-update;com.github.spotbugs:spotbugs-annotations;external_dependency} -->
+      <version>4.8.3</version> <!-- {x-version-update;com.github.spotbugs:spotbugs-annotations;external_dependency} -->
```

Comment:
@lmolkova is this a bug with this being a runtime dependency?

Reply:
I'm pretty sure it's a bug. @heyams could you confirm?

Reply:
It was like that for 2 years. You can move it down to be part of test dependencies? or i can make that change later after you PR is merged.